### PR TITLE
Olwa dev 1

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

File: src/com/hms/flexyosisoftconnector/Main.java [hunk 1]
```diff
@@ -109,10 +110,12 @@ private static void getNewPayloadToSend() {
       }
     } catch (IOException e) {
       Logger.LOG_EXCEPTION(e);
+      Logger.LOG_WARN("Exception thrown when obtaining new batch of datapoints. Will retry.");
```
Comment (alexjhawk) [inline src/com/hms/flexyosisoftconnector/Main.java:113]: inconsistent references to data points. Here you refer to them as "datapoints" but above as "data points"

File: src/com/hms/flexyosisoftconnector/Main.java [hunk 2]
```diff
@@ -109,10 +110,12 @@ private static void getNewPayloadToSend() {
       }
     } catch (IOException e) {
       Logger.LOG_EXCEPTION(e);
+      Logger.LOG_WARN("Exception thrown when obtaining new batch of datapoints. Will retry.");
     }
 
     // if there are data points to send
     if (queuePoints.size() != 0) {
+      Logger.LOG_DEBUG("Grabbed " + queuePoints.size() + " datapoints for proccessing.");
```
Comment (alexjhawk) [inline src/com/hms/flexyosisoftconnector/Main.java:118]: inconsistent datapoint reference

Comment (oliver-walker-hms) [reply src/com/hms/flexyosisoftconnector/Main.java:118]: i've been trying to use datapoint everywhere and keep slipping up on that one

Comment (oliver-walker-hms) [reply src/com/hms/flexyosisoftconnector/Main.java:119]: fixed

**Comment by TomKimsey** [comment]
Careful force pushing during a review.

**Pull request merged**